### PR TITLE
[bitnami/*] Fix parameters for schema generation

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/kubeapps/values.yaml [hunk 1]
```diff
@@ -1515,7 +1515,7 @@ clusters:
     domain: cluster.local
 ## @param featureFlags [object] Feature flags (used to switch on development features)
 ##
-featureFlags:
+featureFlags: ""
```
Comment (antgamdia) [inline bitnami/kubeapps/values.yaml:1518]: Wouldn't it be `featureFlags: {}` instead since it is an object? cc @miguelaeh

Comment (carrodher) [reply bitnami/kubeapps/values.yaml:1518]: You're totally right

Comment (miguelaeh) [reply bitnami/kubeapps/values.yaml:1518]: ups, yes, my fault

**Pull request merged**